Assets/_GameJamCat/Scripts/CatBehaviour.cs

```diff
@@ -31,11 +36,61 @@ public void Initialize(Vector3 spawnPosition, Vector3 scale)
         private void Awake()
         {
             CatRenderer = GetComponentInChildren<Renderer>();
+            _playableDirector = GetComponent<PlayableDirector>();
         }
 
-        private void Update()
+        private void Start()
         {
+            SetupVirtualCams();
+        }
+
+        /// <summary>
+        /// Fetches main cam to get cinemachine brain and player cam so we can transition from player cam to cat cam
+        /// </summary>
+        private void SetupVirtualCams()
+        {
+            foreach (var output in _playableDirector.playableAsset.outputs)
+            {
+                if (output.streamName == "Dialouge")
```

Comment:
put "Dialogue" as a var on top

same with player

Reply:
Also fix typo in dialogue lol